### PR TITLE
Make project more contributor friendly

### PR DESCRIPTION
**Pull request by halaney**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -91,6 +91,7 @@ mod tests {
             chip
         }
 
+        // Decrements timers if not 0
```
Comment (MattCatz) [inline src/lib.rs]: What do these do? What happens when they get to zero?

File: src/lib.rs [hunk 2]
```diff
@@ -100,12 +101,14 @@ mod tests {
             }
         }
 
+        // Loads a buffer into memory at location 0x200
```
Comment (MattCatz) [inline src/lib.rs]: Why the location 0x200?

File: src/lib.rs [hunk 3]
```diff
@@ -100,12 +101,14 @@ mod tests {
...
         pub fn load_rom(&mut self, buffer: Vec<u8>) {
             for (i, data) in buffer.iter().enumerate() {
                 self.memory[0x200 + i] = *data;
             }
         }
 
+        // Loads the fontset chip 8 provides into memory
```
Comment (MattCatz) [inline src/lib.rs]: Is this redundant to line 113?

File: README [hunk 4]
```diff
@@ -0,0 +1 @@
+A chip-8 emulator written in rust
```
Comment (MattCatz) [inline README]: Maybe put your low hanging fruit list here too.

File: src/lib.rs [hunk 5]
```diff
@@ -93,8 +93,9 @@ impl<'a> Chip8<'a> {
         chip
     }
 
-    // Decrements timers if not 0
-    fn update_timers(&mut self) {
+    // Decrements timers at 60 Hz if not 0
```
Comment (halaney) [inline src/lib.rs]: Good thing you commented here, I am working off memory and not the docs which is not good. Unit testing this thing will surely end up revealing multiple bugs

**Pull request merged**